content/docs/guides/uninstall.md

```diff
@@ -124,24 +125,43 @@ namespace "<namespace>" deleted
 
 Repeat the steps above for each mesh installed in the cluster. After there are no OSM control planes remaining, move to following step.
 
-## Removal of OSM Cluster Wide Resources
+## OSM Resources Left Behind After Uninstalling OSM from Unmanaged and Managed Environments
 
-OSM ensures that all the CRDs mentioned [here](https://github.com/openservicemesh/osm/tree/{{< param osm_branch >}}/cmd/osm-bootstrap/crds) exist in the cluster at install time. If they are not already installed, the `osm-bootstrap` pod will install them before the rest of the control plane components are running. This is the same behavior when using the Helm charts to install OSM as well. Uninstalling OSM will only remove/un-patch the conversion webhook fields from all the CRDs (which OSM adds to support multiple CR versions) and will not deleted them for primarily two reasons: 1. CRDs are cluster-wide resources and may be used by other service meshes running in the same cluster, 2. deletion of a CRD will cause all custom resources corresponding to that CRD to also be deleted. 
+Uninstalling OSM (through the `osm uninstall mesh` command or through Helm) will uninstall osm control plane components.
+However, it will leave behind certain OSM resources to prevent unintended consequences for the cluster after uninstalling OSM.
+The resources that are left behind will depend on whether OSM was uninstalled from a managed or unmanaged cluster environment.
 
-If there are no other service meshes running in the same cluster and the required custom resources have been backed up, the CRDs can be removed from the cluster using `kubectl`.
+`osm uninstall mesh` uninstalls the following resources in both unmanaged and managed environments:
+1. remove osm control plane components, including control plane pods,
+2. remove/un-patch the conversion webhook fields from all the CRDs (which OSM adds to support multiple CR versions),
```

Comment:
I think this reads a bit better since the sentences start with "remove" rather than a list of resources, but just a suggestion. Also removing commas for consistency since we don't use them in other lists in the repo
```suggestion
`osm uninstall mesh` does the following in both unmanaged and managed environments:
1. removes OSM control plane components, including control plane pods
2. removes/un-patches the conversion webhook fields from all the CRDs (which OSM adds to support multiple CR versions)
```